<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Исследование автокорреляционной функции</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Signal</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>351</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Form</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Sine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cosine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Triangle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sawtooth</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Amplitude</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCustomPlot" name="signalPlot" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Noise</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>351</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Amplitude</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCustomPlot" name="noisePlot" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="QCustomPlot" name="noisePlot_2" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Summary</string>
        </property>
        <widget class="QPushButton" name="runButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>110</y>
           <width>92</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>351</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Number of points</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="numberOfPoints">
            <property name="minimum">
             <number>64</number>
            </property>
            <property name="maximum">
             <number>10240</number>
            </property>
            <property name="value">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Correlation lag</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="maximum">
             <number>1024</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
